<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QWidget" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QFrame" name="borderFrame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="borderButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>边框颜色</string>
   </property>
  </widget>
  <widget class="QFrame" name="highlightFrame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>54</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="highlightButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>54</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>高亮颜色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>98</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="sxRadio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>102</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
    </font>
   </property>
   <property name="text">
    <string>顺</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sjRadio">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>102</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
    </font>
   </property>
   <property name="text">
    <string>随</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dqRadio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>102</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
    </font>
   </property>
   <property name="text">
    <string>单</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>borderButton</tabstop>
  <tabstop>highlightButton</tabstop>
  <tabstop>sxRadio</tabstop>
  <tabstop>sjRadio</tabstop>
  <tabstop>dqRadio</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
